webapp/packages/plugin-data-spreadsheet-new/src/DataGrid/DataGridTable.tsx

```diff
@@ -356,6 +256,22 @@ export const DataGridTable = observer<IDataPresentationProps>(function DataGridT
     }
   };
 
+  useEffect(() => {
+    const gridDiv = dataGridDivRef.current;
+
+    if (
+      gridDiv &&
+      innerState.lastCount > model.source.count &&
+      model.source.count * rowHeight < gridDiv.scrollTop + gridDiv.clientHeight - headerHeight
+    ) {
+      gridDiv.scrollTo({
+        top: model.source.count * rowHeight - gridDiv.clientHeight + headerHeight - 1,
+      });
+    }
```

Comment:
The grid will scroll automatically to the active selected cell. We should not repeat this functionality